tools/measurement/coverage/coverage.gradle

```diff
@@ -21,7 +21,11 @@ task generateCoverageMeasurements(type: GenerateMeasurementsTask) {
     description 'Runs checkCoverage task in all projects and calculates coverage percents.'
     group 'Measurements'
 
-    dependsOn rootProject.allprojects.collect { it.tasks.withType(JacocoReport) }.flatten()
+    dependsOn rootProject.allprojects.findAll {
+        it.path.startsWith(':firebase-')
```

Comment:
The intention here is to only depend on check tasks of firebase products instead of all. Is there a more reliant way to distinguish firebase products from other utility subprojects?

Reply:
You can check if the projects are using the Firebase library plugin. This plugin encapsulates some of the release and testing logic, and we only use it on the actual Firebase projects.

`project.extensions.findByType(FirebaseLibraryExtension.class) != null` should do the trick.

Reply:
Looks like all below projects have this plugin:
```
project ':firebase-common'
project ':firebase-database'
project ':firebase-database-collection'
project ':firebase-datatransport'
project ':firebase-firestore'
project ':firebase-functions'
project ':firebase-inappmessaging-display'
project ':firebase-storage'
project ':protolite-well-known-types'
project ':firebase-common:ktx'
project ':firebase-firestore:ktx'
project ':firebase-functions:ktx'
project ':transport:transport-api'
project ':transport:transport-backend-cct'
project ':transport:transport-runtime'
```
by running
```
rootProject.allprojects.findAll { it.extensions.findByType(FirebaseLibraryExtension) != null }.forEach { println it }
```

Anyway to exclude `:protolite-well-known-types`, `:transport:transport-api`, `:transport:transport-backend-cct`, and `:transport:transport-runtime`?

Reply:
I think we should include those. The transport ones are part of Firelog. Any reason for excluding?

Reply:
Done. Discussed offline.